FlowCrypt/Controllers/Inbox/InboxViewController.swift

```diff
@@ -370,9 +370,13 @@ extension InboxViewController {
 
         switch error {
         case GmailServiceError.invalidGrant:
-            appContext.globalRouter.renderMissingPermissionsView(
-                appContext: appContext
-            )
+            do {
+                try appContext.globalRouter.renderMissingPermissionsView(
+                    appContext: appContext
+                )
+            } catch {
```

Comment:
We can move this `do/catch` block inside `renderMissingPermissionsView` to remove code duplication. So it'll look like
```swift
do {
    let email = try appContext.encryptedStorage.activeUser?.email ?? ""
   ...
} catch {
    keyWindow.rootViewController?.showAlert(....
}
```

Reply:
Sounds good 👍